tests/test_lens_pop.py

```diff
@@ -150,6 +152,7 @@ def test_supernovae_lens_pop_instance():
     assert pes_lens_class._source_type == "point_source"
     assert "z" in pes_lens_class.source.source_dict.colnames
     assert len(pes_lens_class.source.source_dict) == 1
+    assert pes_lens_pop.factor == 1
```

Comment:
I don't really see a test of the new feature for the large_skyarea being different than the sky_are, hence I don't see the feature being tested